internal/compute/digitalocean_test.go

```diff
@@ -51,48 +55,46 @@ func TestNewDigitalOceanProvider(t *testing.T) {
 			// Check provider
 			assert.NoError(t, err)
 			assert.NotNil(t, provider)
-			assert.NotNil(t, provider.doClient)
 		})
 	}
 }
 
 func TestDigitalOceanProvider_ValidateCredentials(t *testing.T) {
```

Comment:
logic of test needs to be updated to actually be testing the validate func. right now it is too mocked.